contents/docs/user-guides/organizations-and-projects.md

```diff
@@ -0,0 +1,90 @@
+
+Every new organization (including the one created for you on account creation) comes with a fresh project named "Default Project". You can rename or delete it as you see fit.
+
+To switch between projects, navigate to project settings, or to create new projects, use the project switcher in the middle of the top bar. You can also quickly go to the current project's settings from the sidebar.
+
+> **Note:** Multiple projects within an organization belong to our premium team-oriented offering. To use this feature, [set up PostHog Cloud billing](https://app.posthog.com/organization/billing) or [contact us for a self-hosted license](/pricing).
+
+### Ways of organizing projects
+
+There are several practical ways of using projects:
+
+- A project per environment – you may want to test out analytics instrumentation in development or staging environments, while keeping that test data separate from production.
+- A project per product – if you have several unlinked products, it makes sense to reflect that separation in PostHog too.
+- A project per multiple products or the whole company – if your offering is interlinked or you want to track the user journey holistically (e.g. how many company blog readers convert to paid product users), you can use a single project for all digital assets.
+
+Which way is the best fit for you depends on the characteristics of your product and business.
+
+## Permissions
+
+By default all organization members have access to all its projects. Their permissions are then dicated by their organization-wide access level.
+
+This makes for the smoothest collaboration. However, if you require access to data to be more granular, see [private projects](#private-projects).
+
+### Access levels
+
+There are three access levels in PostHog: Member, Administrator, and Owner.
+Please note that an organization must have one and only one Owner.
+
+See permissions at each level below:
+
+| | Member (base level) | Administrator | Owner |
+| --- | --- | --- | --- |
+| Viewing and querying project data | ✅ | ✅ | ✅ |
+| Inviting new members | ✅ | ✅ | ✅ |
+| Billing management | ✅ | ✅ | ✅ |
+| Project creation and deletion | ❌ | ✅ | ✅ |
+| Project settings management (incl. [project-specific memberships](#private-projects)) | ❌ | ✅ | ✅ |
+| Organization settings management (incl. memberships) | ❌ | ✅ | ✅ |
+| Leaving the organization | ✅ | ✅ | ❌ |
+| Transferring ownership | ❌ | ❌ | ✅ |
+| Organization deletion | ❌ | ❌ | ✅ |
+
+Access levels can be viewed and changed in the Members section of organization settings.
+
+### Private projects
+
+If you'd like to restrict access to data within the organization to only those who need it, you can make a project _private_. Projects with this setting enabled become **secret by default and invite-only** (except for Administrators and the Owner, who get access implicitly).
```

Comment:
```suggestion
If you'd like to restrict access to data within the organization to only those who need it, you can make a project _private_. Projects with this setting enabled become **private by default and invite-only** (except for Administrators and the Owner, who get access implicitly).

I'm sorry, I know I made this suggestion but just realized secret implies that it's completely hidden from other members, but it's not secret (other team members can still see it exists and its name), it's private
```

Reply:
Ah, I thought it was a conscious suggestion to make them secret in the product too, but very well, let's just keep them private as currently implemented.

Reply:
I think it's fine to keep it like this, we can get more feedback from users in the upcoming weeks as to whether there's a use case for secret projects, but my guess is that it won't.